sea-orm-cli/Cargo.toml

```diff
@@ -29,7 +29,9 @@ sea-schema = { version = "^0.2.4", default-features = false, features = [
 sqlx = { version = "^0.5", default-features = false, features = [ "mysql", "postgres" ] }
 
 [features]
-default = [ "runtime-async-std-native-tls" ]
+# Warning: removing this line means one cannot simply run `cargo install sea-orm-cli`.
+# They have to run `cargo install sea-orm-cli --features runtime-async-std-native-tls`
+# default = [ "runtime-async-std-native-tls" ]
```

Comment:
sea-orm-cli is an app. People shouldn't care about which runtime we are using.
I think we should ship a default runtime.
If it's so problematic to be in the same workspace, we can split it out to its own workspace, or even another repo.

Reply:
Agree, split it out to its workspace worked!

Reply:
Cool. Let's try.